internal/frontend/proto/tesseral/frontend/v1/frontend.proto

```diff
@@ -743,3 +750,46 @@ message DeleteAPIKeyRoleAssignmentRequest {
 }
 
 message DeleteAPIKeyRoleAssignmentResponse {}
+
+message ListAuditLogEventsRequest {
+  // The maximum number of events to return.
+  int32 page_size = 1;
+
+  // A page token, received from a previous `ListAuditLogEvents` call.
+  string page_token = 2;
+
+  // Filters for the results.
+  message Filter {
+    // The start time.
+    google.protobuf.Timestamp start_time = 1;
+
+    // The end time.
+    google.protobuf.Timestamp end_time = 2;
+
+    // The event names to filter by, if any.
+    repeated string event_name = 3;
+
+    // The user ID to filter by, if any.
+    optional string user_id = 4;
+
+    // The session ID to filter by, if any.
+    optional string session_id = 5;
+
+    // The API key ID to filter by, if any.
+    optional string api_key_id = 6;
+  }
+
+  // Filters for the results.
+  //
+  // May only be provided on the initial call. Subsequent page
+  // requests will use the initial filter.
+  Filter filter = 3;
```

Comment:
just inline these as direct params, no nested message

Reply:
maybe we can prefix each of the inlined properties as `filter_...`

Reply:
I've replaced with `filter_...`. Lmk how that looks